UPGRADE-4.0.md

```diff
@@ -1,2 +1,5 @@
 UPGRADE FROM 3.x to 4.0
 =======================
+
+## Admin
+The ``isCurrentRoute()`` method was added to the AdminInterface. You can easily determine which action is the current request.
```

Comment:
* Only one backtick
* Inline code for `AdminInterface`
* Remove "You can easily determine..." setence
* Rebase again against master and complete this part: https://github.com/sonata-project/SonataAdminBundle/blob/master/UPGRADE-4.0.md#admin